backend/middlewares/validators/util.ts

```diff
@@ -46,6 +48,10 @@ export const validateFileType = (mimetype: string): boolean => {
   return allowableContentTypes.has(mimetype);
 };
 
+export const validateImageType = (mimetype: string): boolean => {
```

Comment:
nit: mimetype to media type? https://en.wikipedia.org/wiki/Media_type

Reply:
does it make sense to keep the term `mimetype` since graphql's `FileUpload` uses that naming?

Reply:
oh probably yes then LOL nvmd